### ISSUE TITLE
Exception thrown in door.js if encoding is not supported by Node

### ISSUE THREAD

**Issue by NuSkooler** [NuSkooler/enigma-bbs#23]
Due to recent versions of Node preventing iconv-lite `extendNodeEncodings()`, an exception is thrown when door.py attempts to call `setEncoding()` if the encoding is not supported by Node itself (e.g. 'cp437')

Stack:
```
< string_decoder.js:24
<     throw new Error('Unknown encoding: ' + encoding);
<           ^
< Error: Unknown encoding: cp437
<     at assertEncoding (string_decoder.js:24:11)
<     at new exports.StringDecoder (string_decoder.js:38:3)
<     at Socket.Readable.setEncoding (_stream_readable.js:206:33)
<     at Terminal.setEncoding (/home/bashby/dev/enigma-bbs/node_modules/ptyw.js/lib/pty.js:288:17)
<     at Door.run (/home/bashby/dev/enigma-bbs/core/door.js:50:7)
<     at AbracadabraModule.runDoor (/home/bashby/dev/enigma-bbs/mods/abracadabra.js:157:16)
<     at AbracadabraModule.finishedLoading (/home/bashby/dev/enigma-bbs/mods/abracadabra.js:178:7)
<     at complete (/home/bashby/dev/enigma-bbs/mods/abracadabra.js:126:11)
<     at /home/bashby/dev/enigma-bbs/node_modules/async/lib/async.js:721:13
<     at _toString (/home/bashby/dev/enigma-bbs/node_modules/async/lib/async.js:52:16)

```

**Comment by NuSkooler** [comment]
https://github.com/iiegor/ptyw.js/issues/4

**Comment by NuSkooler** [comment]
This is resolved with c7a6e6f. Possibly temporary solution, but it works.
